src/components/Section/Foreign/Business/SponsorshipItem.jsx

```diff
@@ -145,6 +146,7 @@ export default class SponsorshipItem extends ValidationElement {
   }
 
   render() {
+    const sponsorDOBMinDate = pickDate([this.props.applicantBirthdate, this.props.Birthdate])
```

Comment:
Same here, it doesn't look like `this.props.applicantBirthdate` is ever passed in to this component.

Reply:
![screen shot 2018-11-19 at 2 03 18 pm](https://user-images.githubusercontent.com/1449852/48728838-19b33780-ec04-11e8-9d64-dc4a50e909e4.png)